src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  status: string;
+  onStatus: React.Dispatch<React.SetStateAction<string>>;
+  amountOfItems: number;
+};
+
+export const Footer: React.FC<Props> = ({
+  status,
+  onStatus,
+  amountOfItems,
+}) => {
+  const FILTERS = {
+    all: 'all',
+    completed: 'completed',
+    active: 'active',
+  };
```

Comment:
Great! It would be even better if you allocated a separate file for this type so that it can be used in App.tsx